_config.yml

```diff
@@ -370,3 +370,7 @@ authors:
     name: Jaakko Saha
     email: 6a19b5eb0335707f33dfeb263649da77
     url: https://www.linkedin.com/in/jaakko-saha-490255107/
+  sergei.vorobyov:
+    name: Sergei Vorobyov
+    email: e196ababec915b5e5100983d30f84253
+    url: https://cv.solita.fi/?name=Sergei+Vorobyov%3B
```

Comment:
cv.solita.fi is an internal service the public does not have access to

Reply:
The url: line is removed